src/modules/SX126x/SX126x.cpp

```diff
@@ -1581,7 +1585,11 @@ int16_t SX126x::fixPaClamping() {
   RADIOLIB_ASSERT(state);
 
   // update with the new value
-  clampConfig |= 0x1E;
+  if (enable)
+    clampConfig |= 0x1E;
```

Comment:
Where are these numbers coming from?

Reply:
See comment a few lines up:

```
  // fixes overly eager PA clamping
  // see SX1262/SX1268 datasheet, chapter 15 Known Limitations, section 15.2 for details
```

Which points to:

![image](https://user-images.githubusercontent.com/194491/209332522-c6ede131-234a-4b55-8fbd-5764bf0ff186.png)

Reply:
I meant the rest of it (highlighted incorrect part, sorry):

```c
  // update with the new value
  if (enable)
    clampConfig |= 0x1E;
  else
    clampConfig = (clampConfig & ~0x1E) | 0x80;
```

This seems to imply that if set to false, it will undo the workaround. I don't understand why it sets the MSB though, the datasheet doesn't mention that.

Reply:
> This seems to imply that if set to false, it will undo the workaround. I don't understand why it sets the MSB though, the datasheet doesn't mention that.

Ah, I think I made mistake there, should have been 0x08 (which is "0100" for bits 4-1). I'll fix that.

Reply:
Resolved in feede0b2fce9207133be4af68267377fec7b3ffe (I also slightly clarified the comment).